helpers.go

```diff
@@ -47,10 +47,15 @@ func BuildMultiInsert(tbl string, cols Columns, rows Rows) (Query, error) {
 		args []interface{}
 	)
 
+	if len(cols) == 0 || len(rows) == 0 {
+		return Query{}, errors.New("Rows and Cols can't be empty")
```

Comment:
Are you seriously? There no capitalized or punctuation and new line.
Also my golint say nothing for this line.
`golint helpers.go 
helpers.go:10:2: exported type Column should have comment or be unexported
helpers.go:11:2: exported type Columns should have comment or be unexported
helpers.go:12:2: exported type Row should have comment or be unexported
helpers.go:13:2: exported type Rows should have comment or be unexported
helpers.go:14:2: exported type Array should have comment or be unexported
helpers.go:17:1: exported function NewHttpTransport should have comment or be unexported
helpers.go:17:6: func NewHttpTransport should be NewHTTPTransport
helpers.go:21:1: exported function NewConn should have comment or be unexported
helpers.go:33:1: exported function NewQuery should have comment or be unexported
helpers.go:40:1: exported function BuildInsert should have comment or be unexported
helpers.go:44:1: exported function BuildMultiInsert should have comment or be unexported
`